extensions/cornerstone/src/init.js

```diff
@@ -102,6 +102,25 @@ export default function init({ servicesManager, configuration }) {
     csTools.BrushTool,
   ];
 
+  /* Measurement Service Events */
+  cornerstone.events.addEventListener(
+    cornerstone.EVENTS.ELEMENT_ENABLED,
+    event => {
+      MeasurementService.subscribe(
+        MeasurementService.getEvents().MEASUREMENT_ADDED,
```

Comment:
`getEvents()` because we expect something to register new events?

Reply:
Registering new events is possible in this branch but maybe it's not our case so I can remove it. Still, we have to find a good way to access the EVENTS constant or static prop because in this case, this passed in MeasurementService is an instance already.